floodviz/static/js/hydrograph.js

```diff
@@ -40,8 +45,9 @@ d3.json("../static/data/hydrograph_data.json", function(error, data) {
     // Loop through each symbol / key
     dataNest.forEach(function(d) {
 
-        svg.append("path")
-            .attr("class", "line")
+        svg.append("g")
+            .attr('class', 'rivers')
```

Comment:
CSS rules will be shared by all visualizations, so "rivers" probably isn't the best class name because it sounds like map styling, and because these paths represent gages, not necessarily rivers.